### DISCUSSION TITLE
Small bug with Playbook Converter

### DISCUSSION THREAD

**Discussion by Sahale** [Security-Onion-Solutions/securityonion#5624] [Unsupported Versions] [answered]
Hi.
While creating ElasticSearch Query from Sigma with converter find out that I have to manually edit result because elastic is case sensitive.
Part of my sigma query:
`detection:
    selection:
        EventID: 4656
        ObjectName: 
            - '*.kdbx'`
result is:
`winlog.event_data.ObjectName.security:(*.kdbx)`. This result doesn't return anything.
Correct result: `winlog.event_data.objectName.security:(*.kdbx)`

If use `objectName` in sigma, then result is incorrect at all: `objectName.security:(*.kdbx)`

I think it also touches objectClass and objectType, but I didn't check it.

**Comment by defensivedepth** [comment, marked as answer]
Yes, this looks like an additional mapping we need to add:
ObjectName --> winlog.event_data.objectName

correct?

**Reply by Sahale** [reply]
Hello!
This mapping already exists [here](https://github.com/Security-Onion-Solutions/securityonion-image/blob/master/so-soctopus/so-soctopus/playbook/securityonion-baseline.yml), but with capital "O": `ObjectName: winlog.event_data.ObjectName`
As I understand, it should be edited to `ObjectName: winlog.event_data.objectName`

**Reply by defensivedepth** [reply]
Ok, will get this fixed, thanks for reporting it!

**Comment by defensivedepth** [comment]
@Sahale How were these logs shipped into Security Onion? Winlogbeat, Osquery or Wazuh?

**Reply by Sahale** [reply]
Wazuh

**Reply by PatRussell87** [reply]
I've noticed that Wazuh ships lowercase for the first character, while winlogbeat ships upper case.

The existing mapping will work for Winlogbeat, but not Wazuh without some editing.

Currently creating specific alerts for my environment depending on what endpoint log shipper I'm using.

**Reply by defensivedepth** [reply]
@PatRussell87 yep that is exactly the issue I was trying to figure out. The goal is that all windows eventlogs end up with the same consistent fields no matter your shipper, so I would consider this bug. I will take a look at how this can be fixed.

**Reply by defensivedepth** [reply]
Looking into this further, there are some related things that need to be completed before I can push the fix for this. Targeting next release.

**Comment by defensivedepth** [comment]
@PatRussell87 @Sahale We have a fix for this, just merged. It will be in the next release:

https://github.com/Security-Onion-Solutions/securityonion/pull/6816
